impeller/entity/contents/solid_rrect_blur_contents.cc

```diff
@@ -48,13 +49,19 @@ Color SolidRRectBlurContents::GetColor() const {
   return color_;
 }
 
+namespace {
+Scalar ScaleSigma(Scalar sigma) {
```

Comment:
I thought that the fast path blur was already being scaled by the fact that it was operating in untransformed coordinates...?

Reply:
That math comes from https://github.com/flutter/engine/pull/47405.  It's in the docstring for the function.  This is to make sure that the outputs are the same.